CONTRIBUTION.md

```diff
@@ -32,12 +32,13 @@ Thanks for contributing!
 ### Format of the pull request (PR)
 
 - The PR header for a feature should be prefixed with the Jira issue. Example: `WINC-123:`
-- The PR header for a bug should be prefixed with the Bugzilla number. Example: `Bug 123:`
-- Correctly prefixing the PR header will automatically associate the PR with the Jira issue or Bugzilla bug.
-  - In the case of bug fix PRs, this will also automatically transition the associated Bugzilla bug.
+- The PR header for a bug should be prefixed with the Jira Bug identifier. Example: `OCPBUGS-123:`
+- Correctly prefixing the PR header will automatically associate the PR with the Jira issue or bug.
+  - In the case of bug fix PRs, this will also automatically transition the associated Jira bug.
     - Opening the PR: `ASSIGNED` --> `POST`
     - PR merges: `POST` --> `MODIFIED`
     - `MODIFIED` to the `ON_QA` transition will have to be done manually by the PR author.
+    - PR closes without merging the Jira bug transition back to NEW status
```

Comment:
could you put a colon after `merging`?

Reply:
transition -> transitions

Reply:
Addressed in [2018e78 ](https://github.com/openshift/windows-machine-config-operator/commit/2018e787a8f9ed0b6fa45e4e371708b49cfe6acf)